src/api/routes/summaries.ts

```diff
@@ -17,7 +17,10 @@ const AI_MINUTES_SUMMARY_KEY = "智能纪要";
 export interface SummaryDescriptor {
   groupId: string,
   transcriptId: string,
-  summaryKey: string,
+  summaryDetails: {
+    summaryKey: string;
+    speakerInfo: string;
+  },
```

Comment:
leave `summaryKey` as is. don't add `summaryDetails`. only add a new field called `speakerStats` of the same type as the return value of `getSpeakerStats`.